src/parser/jobs/ast/modules/oGCDs.tsx

```diff
@@ -0,0 +1,125 @@
+
+	private COOLDOWNS_TRACKED = [
+		//dps actions
+		{
+			cooldowns: [this.data.actions.DIVINATION],
+			weight: WEIGHTS.DPS,
+			type: TYPES.DPS,
+			content: <Trans id="ast.ogcds.divination.description">
+				Consider using <DataLink action="DIVINATION" /> more frequently. <DataLink action="DIVINATION" /> provides an Astrologian with a strong amount of raid DPS.
+			</Trans>,
+		},
+		{
+			cooldowns: [this.data.actions.MACROCOSMOS],
+			weight: WEIGHTS.DPS,
+			type: TYPES.DPS,
+			content: <Fragment><Trans id="ast.ogcds.macrocosmos.description">
+				Consider using <DataLink action="MACROCOSMOS" /> more frequently. Frequent uses can heal and inflict a large amount of damage over the course of a fight, potentially resulting in fewer required healing GCDs and a shorter overall fight.
+			</Trans></Fragment>,
+		},
+
+		//healing actions
+		{
+			cooldowns: [this.data.actions.EXALTATION],
+			weight: WEIGHTS.HEALS,
+			type: TYPES.HEALS,
+			content: <Fragment><Trans id="ast.ogcds.exaltation.description">
+				Consider using <DataLink action="EXALTATION" /> more frequently. Frequent uses can heal and mitigate a large amount of damage over the course of a fight, potentially resulting in fewer required healing GCDs. Additionally, <DataLink action="EXALTATION" showIcon={false} /> can provide significant healing and mitigation when timed closely with tank busters.
+			</Trans></Fragment>,
+		},
+		{
+			cooldowns: [this.data.actions.CELESTIAL_INTERSECTION],
+			weight: WEIGHTS.HEALS,
+			type: TYPES.HEALS,
+			content: <Fragment><Trans id="ast.ogcds.celestial.intersection.description">
+				Consider using <DataLink action="CELESTIAL_INTERSECTION" /> more frequently. Frequent uses can mitigate a large amount of damage over the course of a fight, potentially resulting in fewer required healing GCDs.
+			</Trans></Fragment>,
+		},
+		{
+			cooldowns: [this.data.actions.CELESTIAL_OPPOSITION],
+			weight: WEIGHTS.HEALS,
+			type: TYPES.HEALS,
+			content: <Fragment><Trans id="ast.celestial-opposition.suggestion.content">
+				Consider using <DataLink action="CELESTIAL_OPPOSITION" /> more frequently.
+				The heal and regen combined add up to the same potency of a <DataLink action="BENEFIC_II" /> on each player it reaches.
+				Trusting the regens to top off the party HP will save MP and GCDs on healing.
+			</Trans></Fragment>,
+		},
+		{
+			cooldowns: [this.data.actions.NEUTRAL_SECT],
+			weight: WEIGHTS.HEALS,
+			type: TYPES.HEALS,
+			content: <Fragment><Trans id="ast.neutral-sect.suggestion.content">
+				Consider using <DataLink action="NEUTRAL_SECT" /> more frequently.
+				Frequent uses can mitigate a large amount of damage over the course of a fight, potentially resulting in fewer required healing GCDs.
+			</Trans></Fragment>,
+		},
+	]
+	override trackedCds = this.COOLDOWNS_TRACKED
+
+	override addJobSuggestions() {
+		this.COOLDOWNS_TRACKED.forEach(cooldownGroup => {
```

Comment:
I'm not a huge fan of this approach, since it can potentially generate 6 different suggestions if the user is missing targets on all of the skills separately.  Generating multiple suggestions implies to the user that using cooldowns is 6 times more important than something else that generates a single suggestion, and especially to the extent that we usually allow for a lot of not-use on healing cooldowns it seems like a poor signal to send.

I would also prefer to see the suggestion calculations be based on percentUsed (invert the thresholds, basically), in order to match the way the checklist threshold is handled, so that the calculations are the same in both halves of the module, for future maintainer ease of mind.

Reply:
Updated to consolidate healing suggestions since subjective anyway
Thresholds updated to match

Updated suggestion calculation using percentUsed